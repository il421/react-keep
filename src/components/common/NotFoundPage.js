import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
  <div className="not-found">
    <div className="not-found__wrapper">
      <div className="not-found__text">
        Sorry, such a page does not exist in the App. <br /><br />
        Try to find something interesting on the Dashboard page.
      </div>
      <Link to="/" className="not-found__link button">Keep me ...</Link>
    </div>
  </div>
);

export default NotFoundPage;
