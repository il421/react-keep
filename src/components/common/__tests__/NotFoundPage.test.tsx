import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import NotFoundPage from "../NotFoundPage";

test("should render NotFoundPage correctly", () => {
  const wrapper: ShallowWrapper = shallow(<NotFoundPage />);
  expect(wrapper.debug()).toMatchSnapshot();
});
