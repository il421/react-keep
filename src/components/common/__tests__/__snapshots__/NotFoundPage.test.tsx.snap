// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
"<div className=\\"not-found\\">
  <div className=\\"not-found__wrapper\\">
    <div className=\\"not-found__text\\">
      Sorry, such a page does not exist in the App. 
      <br />
      <br />
      Try to find something interesting on the Dashboard page.
    </div>
    <Link to=\\"/\\" className=\\"not-found__link button\\">
      Keep me ...
    </Link>
  </div>
</div>"
`;
